### ISSUE TITLE
create env under Windows + Conda

### ISSUE THREAD

**Issue by terrychan999** [sciwork/swportal#32]
create conda environments: `conda create -n env_name`

get into the environment: `conda activate env_name`

install python and the dependencies:
 - `conda install python=3.7`
 - `conda install -c conda-forge pelican`
 - `conda install markdown`

go to the swportal folder you cloned:
 - `pelican content -s pelicanconf.py`
 - `cd output`
 - `python -m pelican.server`

the are some bugs with `pelican --autoreload --listen`
it will result in OSError: [WinError 87]
ref: https://github.com/getpelican/pelican/issues/2400
so you can only compile website and depoly/publish through the command above now

**Comment by yungyuc** [comment]
Thank you @terrychan999 for reporting this.  Is this a bug in pelican, or something we need to fix in swportal?

**Comment by terrychan999** [comment]
Yes, it's a bug in pelican. I just found out the bug was fixed with https://github.com/getpelican/pelican/pull/2750
But it is now on the master branch, hasn't become a new release version.

**Comment by terrychan999** [comment]
The bug is fixed in the new release version https://github.com/getpelican/pelican/releases/tag/4.5.0

But the installation steps should change a little bit:
Although the version of Pelican on conda-forge is update-to-date
It will cause "condaVerificationError" during install (maybe due to the non-unicode system locale/path incompatible)
So changing the command to "pip install pelican[markdown]" will be better
This will install both Pelican(and its dependencies) and Markdown

If you got any conda dependencies error, do `conda clean --all` may fix your problem
Caution: `conda clean --all` will remove all the packages inside your conda environment

New Steps example:
```
conda create -n env_name
conda activate env_name
conda install python=3.7
pip install "pelican[markdown]"
pelican --autoreload --listen
```

**Comment by yungyuc** [comment]
What should we do with this issue?  Any comments?

**Comment by terrychan999** [comment]
Actually this issue can be closed.
Just like to share a way to set up environment with Windows.

**Comment by yungyuc** [comment]
It may be a good idea to add the description to a file along with README.  How do you think?

**Comment by terrychan999** [comment]
Sounds great. I will make a pull request for it :)
